### PR TITLE
Dev/remove http patch

### PR DESCRIPTION
**Pull request by whitphx**

### CODE REVIEW

File: packages/common/src/requirements.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// stlite-specific requirements
+export const STLITE_REQUIREMENTS = [
+  "urllib3==2.2.0", // `urllib3` supports Pyodide since 2.2.0. https://urllib3.readthedocs.io/en/stable/reference/contrib/emscripten.html
```
Comment (lukasmasuch) [inline packages/common/src/requirements.ts]: nit: can we do >=2.2.0 here?

**Review by LukasMasuch — APPROVED** [review]
LGTM 👍

File: packages/common/src/requirements.ts (hunk 1 above)
Comment (whitphx) [reply packages/common/src/requirements.ts]: Oops correct!

**Comment by whitphx** [comment]
Well.. changed my mind; specifying `urllib3>=2.2.0` in `setup.py` just like what was done for `pyodide-http` is much simpler...

**Review by LukasMasuch — APPROVED** [review]
LGTM 👍

**Pull request merged**